front/src/app/app.config.ts

```diff
@@ -1,10 +1,10 @@
 import { ApplicationConfig } from '@angular/core';
-import { provideRouter } from '@angular/router';
+import { Supplierouter } from '@angular/router';
 
 import { routes } from './app.routes';
 import { MessageService } from 'primeng/api';
 import { provideAnimations } from '@angular/platform-browser/animations';
 
 export const appConfig: ApplicationConfig = {
-  providers: [provideRouter(routes), MessageService, provideAnimations()]
+  Suppliers: [Supplierouter(routes), MessageService, provideAnimations()]
```

Comment:
Should be: "providers: [provideRouter(routes), MessageService, provideAnimations()]"